### PR TITLE
updating DacPac and SC ext to latest versions

### PR DESCRIPTION
**Pull request by udeeshagautam**
This PR updates Dacpac and Schema compare ext. versions to latest (Jumps to 1.0.0) in gallery.

### CODE REVIEW

**Comment by udeeshagautam** [comment]
@kburtram please merge if the changes look okay (since we don't have permissions to merge). Thanks!

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -1272,14 +1272,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.8.0",
+							"version": "1.0.0",
 							"lastUpdated": "10/10/2019",
```
Comment (kisantia) [inline extensionsGallery-insider.json]: the "lastUpdated" date should be updated too

Comment (udeeshagautam) [reply extensionsGallery-insider.json]: Thanks for catching! done.

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -1314,8 +1314,7 @@
 							]
 						}
 					],
-					"statistics": [],
-					"flags": "preview"
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json]: I haven't tested, but I'd leave the flags empty instead of removing it.

Comment (alanrenmsft) [reply extensionsGallery-insider.json]: other non-preview extensions all left it blank.

**Comment by alanrenmsft** [comment]
I'll take care of the merge for you

**Pull request merged**